components/admin_console/billing/billing_subscriptions/index.tsx

```diff
@@ -210,4 +218,15 @@ const BillingSubscriptions = () => {
     );
 };
 
-export default BillingSubscriptions;
+function mapDispatchToProps(dispatch: Dispatch<GenericAction>) {
+    return {
+        actions: bindActionCreators<ActionCreatorsMapObject<ActionFunc| GenericAction>, Props['actions']>({
+            getCloudSubscription,
+            getCloudProducts,
+            getCloudCustomer,
+            getStandardAnalytics
+        }, dispatch),
+    };
+}
+
+export default connect(null, mapDispatchToProps)(BillingSubscriptions);
```

Comment:
i dont think we need to add dispatchToProps to this functional component, since if you observe we are using hooks to get actions and states in this component. so we should follow the same pattern for this component atleast

Reply:
Can you please change that, then we can continue with the review. Please let us know if you face any problem